treeherder/model/fixtures/repository.json

```diff
@@ -2005,5 +2005,32 @@
       "repository_group": 9,
       "tc_root_url": "https://firefox-ci-tc.services.mozilla.com"
     }
+  },
+  {
+    "pk": 142,
+    "model": "model.repository",
+    "fields": {
+      "dvcs_type": "git",
+      "name": "enterprise",
+      "url": "https://github.com/mozilla/enterprise-firefox",
+      "branch": "enterprise-main",
+      "active_status": "active",
+      "codebase": "enterprise-firefox",
+      "repository_group": 6,
+      "tc_root_url": "https://firefox-ci-tc.services.mozilla.com"
+    }
+  },
+  {
+    "pk": 143,
+    "model": "model.repository",
+    "fields": {
+      "dvcs_type": "git",
+      "name": "enterprise-try",
+      "url": "https://github.com/mozilla/enterprise-firefox",
+      "active_status": "active",
+      "codebase": "enterprise-firefox",
+      "repository_group": 6,
+      "tc_root_url": "https://firefox-ci-tc.services.mozilla.com"
+    }
```

Comment:
My intent with this second block is for all tasks that run against pull requests to show in this project. I'm not really sure if this is doing that or not, but looking at some of the older projects (e.g `gaia-try`), I don't see anything special that they had.